File: source/BolaoInfra/Models/Pais.cs
using System;
using System.Collections.Generic;

namespace BolaoInfra.Models
{
    public partial class Pais
    {
        public Pais()
        {
            JogosA = new HashSet<Jogo>();
            JogosB = new HashSet<Jogo>();
        }

        public int CodPais { get; set; }
        public string Nome { get; set; }
        public string Arquivo { get; set; }

        public virtual ICollection<Jogo> JogosA { get; set; }
        public virtual ICollection<Jogo> JogosB { get; set; }
    }
}

File: source/bolao_app/lib/models/pais.dart
class Pais {
  int codPais;
  String nome;
  String arquivo;

  Pais({required this.codPais,
    required this.nome,
    required this.arquivo
  });

  factory Pais.fromJson(dynamic json) {
    return Pais(codPais: json['codPais'] as int,
        nome: json['nome'] as String,
        arquivo: json['arquivo'] as String);
  }
}